### PR TITLE
Copy index.html from is2 2023

### PR DESCRIPTION
**Pull request by scottyhq**
Some of the changes from IS2 2023 were missing (https://github.com/ICESAT-2HackWeek/ICESat-2-Hackweek-2023/commit/602bd954abfdfa628dc6984314b7653e17989aa8). I just copied index.html over.

### CODE REVIEW

**Comment by jomey** [comment]
For IS2, we swapped the sequence of the page to put the about below the schedule once the applications were close. 

Is that something we should keep?
I am torn whether this should be a permanent change.

File: {{ cookiecutter.repo_directory }}/index.html [hunk 1]
```diff
@@ -534,6 +551,17 @@ <h3 class="section-heading text-center mb-3">Our Sponsors</h3>
...
+<!-- //intranet button -->
+<section class="section">
+    <div class="container">
+        <div class="about-cta text-center mb-5">
+            <a class="btn btn-secondary btn-lg  mb-5" href="https://uwnetid.sharepoint.com/sites/og_icesat2_hackweek" target="_blank">
+                Organizing Team Intranet
+            </a>
+        </div>
+    </div>
+</section>
```
[the remark this answers is not in the document]
Comment (jomey) [reply {{ cookiecutter.repo_directory }}/index.html:563]: Could be added to the top navigation if we decide that these will live for a longer time. At the same time, what is the likelihood someone going from an event page to the general Hackweek advertising page. Do they need more convincing :smiley: ?

[the remark this answers is not in the document]
Comment (JessicaS11) [reply {{ cookiecutter.repo_directory }}/index.html:563]: I hope not! 😃

**Comment by JessicaS11** [comment]
> For IS2, we swapped the sequence of the page to put the about below the schedule once the applications were close.

> Is that something we should keep?
> I am torn whether this should be a permanent change.

Good question. Would making "about" collapsible be easy to do and a good compromise? Or adding some sort of "section order" toggle? Trying to think of ways we can have the best of both worlds but just change a single setting somewhere. For that matter, a few "configurations" that are along the lines of "advertising", "applications-open", "applications-closed", "during-event" to manage turning on/off all the buttons/links and order of things?

**Pull request merged**